build-release.sh

```diff
@@ -234,13 +234,15 @@ if [ "${build_mono}" == "1" ]; then
   mkdir -p ${binbasename}_64
   cp out/linux/x64/tools-mono/godot.x11.opt.tools.64.mono ${binbasename}_64/${binbasename}.64
   cp -rp out/linux/x64/tools-mono/GodotSharp ${binbasename}_64/
+  cp -rp out/aot-compilers ${binbasename}_64/GodotSharp/Tools/
```

Comment:
As discussed on Discord, the ios cross-compiler is only usable from macOS, so it should only be included with the macOS editor binary, not Windows/Linux. I'll edit this.